modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -198,14 +218,14 @@ public void start() {
     /**
      * Extracts the local state (if any) and starts the CMG.
      *
-     * @return Future that resolves into the CMG Raft service or {@code null} if the local state is empty.
+     * @return Future, that resolves into the CMG Raft service, or {@code null} if the local state is empty.
```

Comment:
Would they really put that comma after 'Future' in English?

Reply:
I've put these commas to emphasize the fact that the future itself can be null and it will not resolve to a null value